src/Adapter/Module/ModuleDataProvider.php

```diff
@@ -26,21 +26,50 @@
 namespace PrestaShop\PrestaShop\Adapter\Module;
 
 use PrestaShop\PrestaShop\Adapter\Shop\Context;
+use PrestaShop\PrestaShop\Adapter\LegacyContext;
+use Doctrine\ORM\EntityManager;
 
 class ModuleDataProvider
 {
+    private $entityManager;
+
+    public function __construct(EntityManager $entityManager)
+    {
+        $this->entityManager = $entityManager;
+    }
+
     public function findByName($name)
     {
         $result = \Db::getInstance()->getRow('SELECT `id_module` as `id`, `active`, `version` FROM `'._DB_PREFIX_.'module` WHERE `name` = "'.pSQL($name).'"');
         if ($result) {
             $result['installed'] = 1;
             $result['active'] = $this->isEnabled($name);
+
+            $moduleID = (int)$result['id'];
+            $legacyContext = new LegacyContext();
+            $legacyContext = $legacyContext->getContext();
+            $employeeID = (int)$legacyContext->employee->id;
```

Comment:
You can get the employee data directly with the Synfony service.

Here is an exemple: https://github.com/PrestaShop/PrestaShop/blob/a1160ba4a4a9cd3a4ce788140789ff041d82cc9f/src/PrestaShopBundle/Resources/config/services.yml#L182

But will this code work if called from the front office ?